cloud/amazon/route53.py

```diff
@@ -84,6 +84,11 @@
     required: false
     default: 500
     aliases: []
+  private_zone:
+    description: 
+      - When you have both a public and private zone you need to pass in a true value to modify any records in the private zone.
+    required: false
+    default: false
```

Comment:
More clear to say something like "Set this to True to operate on records within a private zone.  False, the default, operates on public zones."

Also need to have a version_added entry to say that this was added in ansible 1.9.